docs/Guide.md

```diff
@@ -108,3 +108,16 @@ const MyComponent = () => {
 Very often, we use [components with presets](./boilerplate/app/components/Components.md) to share styles across our whole app.
 
 Read more about styling in the [Styling](./concept/Styling.md) docs.
+
+## Patching local dependencies
+
+The react-native ecosystem moves quickly and sometimes, you may need to patch a library locally to get it working. This is especially true for libraries that are not well maintained or have not been updated to work with the latest version of React Native. Ignite used to ship with patches (pre-v10) that were applied with `yarn` and `patch-package`, but the boilerplate no longer needs any patches by default.
+
+Igniting an app allows you to choose `npm`, `yarn`, `pnpm`, or `bun` as a package manager, so we don't have configuration in the boilerplate for patches. If you find that you need to patch a library in your app, we recommend the following:
+
+- `npm` - use [`patch-package`](https://www.npmjs.com/package/patch-package).
+- `yarn@1` - use [`patch-package`](https://www.npmjs.com/package/patch-package).
+- `yarn@3` - use [`yarn patch` tool](https://v3.yarnpkg.com/features/protocols#patch).
+- `yarn@4` - use [`yarn patch` tool](https://yarnpkg.com/features/patching).
```

Comment:
you can still use patch package with these versions as well, couldn't you?

Reply:
Actually no (but *maybe* yes with a bunch more configuration)... which is one of the main reasons I still use yarn 1, even though it's slow. yarn berry changed how the `postinstall` script runs so it's not nearly as reliable anymore. https://github.com/yarnpkg/yarn/issues/5476